addons/sale/tests/test_sale_order.py

```diff
@@ -18,6 +18,16 @@ class TestSaleOrder(SaleCommon):
     # Those tests do not rely on accounting common on purpose
     #   If you need the accounting setup, use other classes (TestSaleToInvoice probably)
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner1 = cls.env['res.partner'].create({
+            'name': 'Partner 1',
+        })
+        cls.partner2 = cls.env['res.partner'].create({
+            'name': 'Partner 2',
+        })
```

Comment:
Not really sure it is worth it, but you can create multiple ones at once
```suggestion
        cls.partner1, cls.partner2 = cls.env['res.partner'].create([
            {'name': 'Partner 1'},
            {'name': 'Partner 2'},
        ])
```

Reply:
Oops :D